BabbleApp/camera_widget.py

```diff
@@ -395,11 +395,11 @@ def render(self, window, event, values):
             if self.settings_config.use_calibration == True:
```

Comment:
In the current state of the app, the "Start Calibration" and "Stop Calibration" buttons are disabled even if the app starts with the "Enable Calibration" checkbox set to true